doc-content/enterprise-only/springboot/creating-self-contained-image-proc.adoc

```diff
@@ -24,33 +24,99 @@ mvn install
 ----
 +
 This command builds the KJAR file and places it into the local Maven repository. By default, this repository is located in the `~/.m2/repo` directory.
-. In the `<BUSINESS-APPLICATION>/<BUSINESS-APPLICATION>-service/src/main/resources` directory, add the following properties to your Spring Boot application `application.properties` file:
+. In the `<BUSINESS-APPLICATION>/<BUSINESS-APPLICATION>-service/src/main/resources` directory, add the following property to your Spring Boot application `application.properties` file:
 +
 [source]
 ----
 kieserver.classPathContainer=true
+----
++
+When this property is set to `true`, {KIE_SERVER} uses the class loader used by the container to load KJAR files and their dependencies.
++
+. Complete one of the following actions to ensure that the {KIE_SERVER} loads the necessary KJAR modules:
+** Add the following property to the `application.properties` file:
++
+[source]
+----
 kieserver.autoScanDeployments=true
 ----
 +
+When this property is set to `true`, {KIE_SERVER} scans and deploys all KJAR modules available in the Spring Boot application, whether they are declared programmatically or through the Maven plug-in.
++
+This option is the simplest method to include all KJAR modules. However, it has two drawbacks:
++
+--
+*** The application sets all container IDs and aliases automatically, based on the group, artifact, and version (GAV) of every KJAR module. You cannot set a custom container ID or alias for a KJAR module.
+*** At startup time, the application scans the JAR file and the class path for KJAR modules. Therefore, the duration of startup might be   increased.
+--
++
+If you need to avoid these drawbacks, use one of the other options to configure every KJAR module individually.
++
+** For each of the KJAR modules that you want to include in the service, add the following properties to the `application.properties` file:
```

Comment:
To configure every KJAR module individually, for each of the KJAR modules that you want to include in the service, add the following properties to the application.properties file: